### ISSUE TITLE
Ideas and Update for 1.12.2

### ISSUE THREAD

**Issue by Camawama** [Mine-and-blade-admin/Battlegear2#247]
Alright, so A lot of people are probably wondering when a port for 1.12.2 will come out. I can that you stopped development around the same time that the 1.9 (Combat) update came out, which in that update added their own dual weilding to the game which is probably why you stopped making updates for the mod. You should release a version for 1.12.2 that disables the combat system and overrides it with the BattleGear dual weilding and such. It was such a great mod and I really need it for the Modpack I am working on. Thank you!

**Comment by darthvader45** [comment]
Agreed, 1.9 combat is nice and all, but this mod really spiced things up with a whole new set of weapons to craft and ways to fight that vanilla never had and likely never will have.  A config option (set to true by default) overrides the 1.9 system like Cama says, and can be turned off to revert back to the 1.9 system.
